src/player/Player.php

```diff
@@ -1374,6 +1378,14 @@ protected function processMostRecentMovements() : void{
 			$this->logger->debug("Exceeded movement rate limit, forcing to last accepted position");
 			$this->sendPosition($this->location, $this->location->getYaw(), $this->location->getPitch(), MovePlayerPacket::MODE_RESET);
 		}
+
+		if ($this->openContainerPosition !== null) {
+			$distance = $this->location->distance($this->openContainerPosition);
+
+			if ($distance > self::MAX_DISTANCE_FROM_CONTAINER) {
```

Comment:
Should this distance be configurable by each inventory in the case of plugin customization?